packages/panda-san/src/variables/container.scss

```diff
@@ -1,7 +1,7 @@
-$container-gutters-default: 16px;
+$container-gutters-default: px-to-rem(16px);
```

Comment:
This reason the build is failing is that we can't use `px-to-rem` in the `variables` folder. [Variables](https://github.com/Lemonpeach/panda-san/blob/master/packages/panda-san/src/index.scss#L1) are imported before mixins (where `px-to-rem` is) since mixins may use some of the variables. 

What we need to do instead is to wrap all variable usage in other parts of the framework with `px-to-rem`. Like here in the [container](https://github.com/Lemonpeach/panda-san/blob/master/packages/panda-san/src/layout/container.scss#L3) component.